layersvt/test/test_screenshot.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 The Khronos Group Inc.
+ * Copyright (c) 2023 Valve Corporation
+ * Copyright (c) 2023 LunarG, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Author: Christophe Riccio <christophe@lunarg.com>
+ */
+
+#include <vulkan/vulkan_core.h>
+#include <vulkan/vulkan_beta.h>
+
+#include <gtest/gtest.h>
+#include "layer_test_helper.h"
+
+#include <cstdarg>
+
+static const char* kLayerName = "VK_LAYER_LUNARG_screenshot";
+
+class ScreenshotTests : public VkTestFramework {
+   public:
+    ScreenshotTests(){};
+    ~ScreenshotTests(){};
+
+    static void SetUpTestSuite() {}
+    static void TearDownTestSuite(){};
+};
+
+TEST_F(ScreenshotTests, init_layer) {
+    TEST_DESCRIPTION("Test Creating a Vulkan Instance with a layer");
+
+    layer_test::VulkanInstanceBuilder inst_builder;
+    VkResult err = inst_builder.Init(kLayerName);
+    EXPECT_EQ(err, VK_SUCCESS);
```

Comment:
Actually, it wouldn't be too hard to load the ppm file and validate it. Don't worry about adding it now, just figured I should say that out loud since ppm is a text file more or less.

Reply:
This case is a lot more involved, implying writing enough code to render frame so I left it alone.